save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -259,21 +248,53 @@ class ProjectController(
     @ApiResponse(responseCode = "403", description = "Not enough permission for project deletion.")
     @ApiResponse(responseCode = "404", description = "Either could not find such organization or such project in such organization.")
     fun deleteProject(
+        @PathVariable organizationName: String,
+        @PathVariable projectName: String,
+        @RequestParam status: ProjectStatus,
+        authentication: Authentication
+    ): Mono<StringResponse> =
+            projectService.findWithPermissionByNameAndOrganization(
+                authentication, projectName, organizationName, Permission.DELETE
+            )
+                .filter {
+                    (it.status == ProjectStatus.CREATED && status == ProjectStatus.DELETED) || status == ProjectStatus.BANNED
+                }
+                .switchIfEmptyToNotFound {
+                    "Could not find created project with name $projectName."
+                }
+                .map {
+                    projectService.deleteProject(it, status)
```

Comment:
What's the difference between deletion and ban of project here? We anyway call  `projectService.deleteProject`
Thus, why do we actually need option `ProjectStatus.BANNED`?